src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -1,37 +1,87 @@
+/**
+ * @jest-environment jsdom
+ */
+
 import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle } from '../__fixtures__/icons'
+import { faCoffee, faCircle, faAlien } from '../__fixtures__/icons'
 import { compileAndMount, mountFromProps } from '../__fixtures__/helpers'
 import FontAwesomeIcon from '../FontAwesomeIcon'
 
 beforeEach(() => {
-  library.add(faCoffee, faCircle)
+  library.add(faCoffee, faCircle, faAlien)
 })
 
-test('using array format', () => {
-  const wrapper = mountFromProps({ icon: ['fas', 'coffee'] })
-
-  expect(wrapper.element.tagName).toBe('svg')
-  expect(wrapper.element.classList.contains('fa-coffee')).toBeTruthy()
+afterEach(() => {
+  library.reset()
 })
 
-test('using string format', () => {
-  const wrapper = mountFromProps({ icon: 'coffee' })
+const version6 = process.env.npm_lifecycle_event.includes('test.6');
```

Comment:
`v5` does not support the `long-prefix` (`fa-thin`) or the `long-icon-name format` ('fa-alien`).
I created this to help differentiate between tests that 5 does NOT need to run, which would include any test with a `long-prefix` and/or a `long-icon-name format`.